src/components/Pokemonlist.js

```diff
@@ -0,0 +1,55 @@
+import React, {useEffect, useState} from "react";
+import axios from "axios";
+import Pokemon from "./Pokemon";
+
+function Pokemonlist() {
+    const [limit, setLimit] = useState(20)
+    const [page, setpage] = useState(0)
+    const [pokedex, setPokedex] = useState([])
+
+    useEffect(() => {
+        async function fetchPokedex() {
+            try {
+                const result = await axios.get(`https://pokeapi.co/api/v2/pokemon?limit=${limit}&offset=${(page * 20)}`)
+                setPokedex(result.data)
+            } catch (error) {
+                console.error(error)
+            }
+        }
+
+        void fetchPokedex()
+    }, [page, limit])
+
+    return (
+        <>
+            <div className='outer-nav-container'>
+            <nav>
+                {Math.ceil(pokedex.count / limit)} pages
+                <div className='page-navigation'>
+                    <button type="button" disabled={page < 1 && true} onClick={() => setpage(page - 1)}>last page</button>
```

Comment:
Als je met de setter de huidige waarde wil aanpassen, kun je ook de waarde gebruiken die je meekrijgt. Op deze manier wordt er niet in memory telkens een nieuwe functie gemaakt
```suggestion
                    <button type="button" disabled={page < 1 && true} onClick={() => setpage(currentPage => currentPage - 1)}>last page</button>
```